File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/loading/NonBlockingLoadingActivity.java
package com.jdroid.android.sample.ui.loading;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class NonBlockingLoadingActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return NonBlockingLoadingFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/loading/NonBlockingLoadingActivity.kt
package com.jdroid.android.sample.ui.loading

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class NonBlockingLoadingActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return NonBlockingLoadingFragment::class.java
    }
}
